test/router_test.exs

```diff
@@ -197,4 +197,56 @@ defmodule Kitto.RouterTest do
     assert conn.status == 401
     Application.delete_env :kitto, :auth_token
   end
+
+  test "POST dashboards/:id reloads single dashboard" do
```

Comment:
change the test description to 

"POST /dashboards/:id reloads the specified dashboad"